sceptre/plan/actions.py

```diff
@@ -861,6 +862,12 @@ def _get_cs_status(self, change_set_name):
                 cs_exec_status in ["UNAVAILABLE", "AVAILABLE"]
         ):
             return StackChangeSetStatus.PENDING
+        elif (
+                cs_status == "FAILED" and
+                cs_reason is not None and
+                "submitted information didn't contain changes" in cs_reason
```

Comment:
There's a different message for SAM templates. Take a look here: https://github.com/Sceptre/sceptre/pull/1099/files#diff-47b4a16113b2f85cf1c61bbe71e39c06f5b99e3ac309ddf449c97ecdd79b86aaR558-R574

Reply:
Fixed in #1192